### PR TITLE
refactor(ulid): format test names

### PR DESCRIPTION
**Pull request by timreichen**
Ref: https://github.com/denoland/deno_std/issues/3754

### CODE REVIEW

File: ulid/test.ts [hunk 1]
```diff
@@ -149,7 +149,7 @@ Deno.test("ulid", async (t) => {
   );
 });
 
-Deno.test("monotonicity", async (t) => {
+Deno.test("stubbedUlid() handles onotonicity", async (t) => {
```
Comment (iuioiua) [inline ulid/test.ts:152]:
```suggestion
Deno.test("stubbedUlid() handles monotonicity", async (t) => {
```

**Review by iuioiua — APPROVED** [review]
LGTM. Thank you.

File: ulid/test.ts [hunk 2]
```diff
@@ -149,7 +149,7 @@ Deno.test("ulid", async (t) => {
   );
 });
 
-Deno.test("monotonicity", async (t) => {
+Deno.test("stubbedUlid() handles monotonicity", async (t) => {
```
Comment (kt3k) [inline ulid/test.ts:152]: This indirectly tests `monotonicUlid` (`stubbedUlid` isn't a public API)

Comment (iuioiua) [reply ulid/test.ts:152]: Ah, yes. I'll fix it as I missed it.

Comment (timreichen) [reply ulid/test.ts:152]: ref: https://github.com/denoland/deno_std/pull/4075
Thanks

**Pull request merged**